op-node/p2p/sync.go

```diff
@@ -97,7 +98,8 @@ type syncResult struct {
 type peerRequest struct {
 	num uint64
 
-	complete *atomic.Bool
+	complete   *atomic.Bool
```

Comment:
Is this `complete` field redundant with the information in the `inFlight` map or are they both needed? It seems like we might be able to get rid of this `complete` field and only use the `inFlight` map

Reply:
Removed `peerRequest.complete` field in https://github.com/ethereum-optimism/optimism/pull/10063/commits/b2372bb8eef6a006d8ffbf285c762acce02c6a5d